include/glow/Runtime/Provisioner/Provisioner.h

```diff
@@ -37,7 +37,8 @@ class Provisioner final {
   /// devices. The Provisioner calls the addNetwork method for each
   /// DeviceManager. Returns a GlowErr indicating if the operation was a
   /// success.
-  llvm::Error provision(DAGListTy &networks, Module &module);
+  llvm::Error provision(DAGListTy &networks, Module &module,
+                        CompilationContext &cctx);
```

Comment:
It seems like you only use `backendOpts` out of `cctx` inside `provision()`, right? Do you expect to use other ops from `cctx` in the future? Otherwise might make sense to just pass `backendOpts`.

Reply:
I think theres a reasonable chance we do (otherwise why not just fold backendOpts into the context?). E.g. in @ZchiPitt's log diff we discussed storing the compilation log in the context and one of the downsides was that it wasn't plumbed through in a bunch of places that passed in BackendOpts instead.